widget/playground/src/components/Spacing/Spacing.styles.ts

```diff
@@ -0,0 +1,46 @@
+import { styled } from '@rango-dev/ui';
```

Comment:
Why do we have both 'Spacing' and 'Divider' components? Can we merge them for simplicity?

Reply:
I removed the spacing. however, I don't think adding all these sizes to the divider is interesting.
I used margin in some of the places I handled.

Reply:
You used margin in some cases and divider in others.